modules/targets/darwin/default.nix

```diff
@@ -0,0 +1,51 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  inherit (pkgs.stdenv.hostPlatform) isDarwin;
+  inherit (lib.strings) escapeShellArg;
```

Comment:
Not needed with the `with lib;` line at the top

Reply:
Fixed.